### PR TITLE
Add docs to precompiles constants

### PR DESCRIPTION
**Pull request by iinaki**

### CODE REVIEW

File: src/constants.rs
Comment (juanimedone) [inline src/constants.rs:1]: Probably it should be better to place the description of each precompile right before each function in `precompiles.rs`. You can use `///`.

Comment (juanimedone) [reply src/constants.rs:1]: Yes, good idea.

Comment (iinaki) [reply src/constants.rs:1]: Done!

File: src/precompiles.rs [hunk 1]
```diff
@@ -159,6 +172,8 @@ pub fn modexp(
     Ok(Bytes::copy_from_slice(output))
 }
 
+/// Point addition on the elliptic curve 'alt_bn128'.
```
Comment (juanimedone) [inline src/precompiles.rs:175]:
```suggestion
/// Point addition on the elliptic curve 'alt_bn128' (also referred as 'bn254').
```

File: src/precompiles.rs [hunk 2]
```diff
@@ -217,6 +232,8 @@ pub fn ecadd(
     Ok(Bytes::from(res))
 }
 
+/// Scalar multiplication on the elliptic curve 'alt_bn128'.
```
Comment (juanimedone) [inline src/precompiles.rs:235]:
```suggestion
/// Scalar multiplication on the elliptic curve 'alt_bn128' (also referred as 'bn254').
```

File: src/precompiles.rs [hunk 3]
```diff
@@ -257,6 +274,11 @@ pub fn ecmul(
     Err(PrecompileError::InvalidEcPoint)
 }
 
+/// Elliptic curve pairing operation required in order to perform zkSNARK verification within the block gas limit. Bilinear function on groups on the elliptic curve “alt_bn128”.
```
Comment (juanimedone) [inline src/precompiles.rs:277]:
```suggestion
/// Elliptic curve pairing operation required in order to perform zkSNARK verification within the block gas limit. Bilinear function on groups on the elliptic curve 'alt_bn128' (also referred as 'bn254').
```

**Review by juanimedone — APPROVED** [review]
Just added a little detail, the rest LGTM.

File: src/precompiles.rs (hunk 3 above)
Comment (iinaki) [reply src/precompiles.rs]: Done!

File: src/precompiles.rs (hunk 1 above)
Comment (iinaki) [reply src/precompiles.rs]: Done!

**Comment by juanimedone** [comment]
Now the PR's title can just be 'Add docs to precompiles' or 'Add precompiles' docs'.

**Review by igaray — APPROVED** [review]

**Pull request merged**